tests/data/model-validation.ts

```diff
@@ -940,6 +934,27 @@ type group
       },
     ],
   },
+  {
+    name: "incorrect wildcard restriction should be raised",
+    friendly: `model
+  schema 1.1
+type user
+type group
+  relations
+    define member: [user, user:*:*]
+`,
+    expectedError: [
+      {
+        endColumn: 35,
+        endLineNumber: 6,
+        message: "Invalid syntax",
```

Comment:
Shouldn't the message be more descriptive?

Reply:
See above - this is byproduct of grammar change which requires the types to be specified.